vm/src/slots.rs

```diff
@@ -51,6 +52,7 @@ pub(crate) type CmpFunc = fn(
     PyComparisonOp,
     &VirtualMachine,
 ) -> PyResult<Either<PyObjectRef, PyComparisonValue>>;
+pub(crate) type GetattroFunc = fn(PyObjectRef, PyStrRef, &VirtualMachine) -> PyResult;
```

Comment:
Can the perameters be ref like other slot funcs? Or maybe even &str for second one?

Reply:
Nevermind about second one. It will not  fit to string interning

Reply:
Yeah, I'm planning on having it take a `&PyRef` but `vm.get_attribute()` takes an owned PyObjectRef, so I'll do that big diff separately

Reply:
I didn't work on descr_get because it always get PyObjectRef during get_class_attr but this change also will resolve this issue.